libs/pmp-web/user/single-user-statistics/feature/src/lib/containers/single-user-statistics/single-user-statistics.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="user-statistics__container">
+  <div class="user-statistics__header-container">
+    <h2 class="user-statistics__header-label">Pull requests pending</h2>
+    <pmp-picture-label [label]="mockedUser.name" [picture]="mockedUser.picture"></pmp-picture-label>
+  </div>
+  <div *ngFor="let repo of mockedData">
+    <div class="user-statistics__repository-label">
+      <pmp-picture-label [label]="repo.repository.name" [picture]="repo.repository.picture">
+      </pmp-picture-label>
+    </div>
+    <pmp-single-user-statistics-table [tableData]="repo.data"></pmp-single-user-statistics-table>
```

Comment:
Based on the data you send by the input I believe that it can be generic single-repository-statistics